os/board/rtl8721csm/src/component/os/tizenrt/rtk_netmgr.c

```diff
@@ -304,7 +305,7 @@ void linkup_handler(rtk_reason_t *reason)
 	}
 	argv[0] = data;
 
-	pid = kernel_thread("lwnl80211_cbk_handler", 100, 1024, (main_t)rtk_drv_callback_handler, argv);
+	pid = kernel_thread("lwnl80211_cbk_handler", 100, 2048, (main_t)rtk_drv_callback_handler, argv);
 	if (pid < 0) {
 		vddbg("pthread create fail(%d)\n", errno);

```

Comment:
How about changing the log to `kernel_thread create fail`?
Same as line 333 also.